src/input/mod.rs

```diff
@@ -456,8 +428,17 @@ impl Delta for axis::Key {
         &self,
         input: &Input,
     ) -> Option<TimerDuration> {
-        self.delta(input)
-            .map(|v| v as TimerDuration * input.delta_time())
+        let is_pos = input.state.keys_pressed.contains(&self.pos);
+        let is_neg = input.state.keys_pressed.contains(&self.neg);
+        if is_pos && is_neg {
+            Some(0.0)
+        } else if is_pos {
+            Some(input.delta_time())
```

Comment:
To make situation clear:
The intent behind `timed` was the shortcut for `timed = delta_time * delta`. And with this approach `timed` and `delta` aren't connected anymore. If it's okay for you from design point of view - I'll accept, just to avoid any misunderstanding.

Reply:
I don't understand the intention here, sorry! It looks like the time is scaled by the number of hits in one frame. Unless the event rate and/or frame rate is constant I can see this being a problem...

Reply:
Games update loops are usually designed to be frame-rate independent.

Reply:
Can you please improve docs in this case?

Reply:
Will do.

Reply:
I'm really not sure what to do here. Could you advise?

Reply:
I will take a look a bit later